src/pages/Account.vue

```diff
@@ -7,38 +7,43 @@
 
                 <transition name="fade"><div v-if="errorMessage" v-bind:class="[goodFeedback ? 'good message' : 'message' ]">{{ errorMessage }}</div></transition>
 
+                <div v-if="voucherPayments.length > 0">
```

Comment:
@ob6160 Maybe vue attaches the observer if you observe it but not in normal circumstances?